src/v1.json

```diff
@@ -880,6 +888,21 @@
         }
       }
     },
+    "legacy-oscontainer": {
+      "$id": "#/properties/legacy-oscontainer",
+      "type": "object",
+      "title": "Legacy oscontainer",
+      "required": [
+        "image"
+      ],
+      "properties": {
+        "image": {
+          "$id": "#/properties/legacy-oscontainer/image",
+          "type": "string",
+          "title": "Image"
+        }
+      }
+    },
```

Comment:
using this might cause some issues with consumers of older releases, does that match your understanding?

i.e. they may be expecting this information somewhere else.

Reply:
That is why I kept:

```
meta['images']['legacy-oscontainer'] = {'path': args.name,
                                        'sha256': sha256.hexdigest(),
                                        'size': size_bytes}
```

Without it the updates test break because older releases are looking for it.

Reply:
I think you mean this is why you kept:

```
meta['oscontainer'] = {'image': args.name,
                       'digest': osc_digest}
```

Is that right?

Reply:
yeah

Reply:
Do you have a link to the CI failures? I'd like to dig into this some more.

Reply:
Maybe let's run CI one more time without it and try to disect the failure and maybe try to reproduce locally.

Reply:
Sure thing